CHANGELOG.md

```diff
@@ -15,15 +15,17 @@ All notable changes to this project will be documented in this file.
 - Refactor `RestTemplateConfig` to use `HttpClient 5` and improve proxy handling. (#626)
 - Replace deprecated `connect` with `connectAsync` in `StompClientService`. (#627)
 - Remove redundant blockchain calls to diminish pressure on Ethereum JSON-RPC API. (#632)
-- Harmonize YML internal variables to proper case. (#638)
 - Stop using `TestUtils` in `ContributionServiceTests`. (#640)
+- Merge split URL configuration properties (protocol, host, port) to a single URL field to offer URL validation at startup. (#641)
```

Comment:
This line is duplicated in CHANGELOG. We only need to keep the occurence in the breaking API changes section